### PR TITLE
Autogenerating d.ts files for older Excel and Word versions

### LINKED ISSUE #485: Overview folder needs a toc.yml file
Currently there's no `toc.yml` file in the `overview` folder, which results in there being no TOC, breadcrumbs, or "PDF download" link displayed when the [Overview page is loaded without the `view` parameter in the URL](https://docs.microsoft.com/en-us/javascript/api/overview/overview).

image

We should add a `toc.yml` file to the `overview` folder (just as there's a `toc.yml` file in every other `docs-ref-autogen` subfolder). Maybe just have it contain a link to the Preview or otherwise newest (where there's no Preview) docs page for each host?

image

Adding this `toc.yml` file will eliminate a few errors in the CATS report. Not urgent, but we should do it when we can find time.

### PR DESCRIPTION
**Pull request by AlexJerabek**
Instead of using static d.ts files for the older requirement sets of Excel and Word, this PR generates them using the version-remover tool.

This also fixes #485

### CODE REVIEW

File: generate-docs/tools/dts-diff.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import * as fsx from "fs-extra";
+import { diff } from 'deep-diff';
+import { APISet, parseDTS } from './dts-utilities';
+
+tryCatch(async () => {
+    //fsx.writeFileSync("./tool-inputs/dt-release.d.ts", await fetchAndThrowOnError("https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/master/types/office-js/index.d.ts", "text"));
```
Comment (ElizabethSamuel-MSFT) [inline generate-docs/tools/dts-diff.ts]: just curious about all the commented out stuff...

**Comment by o365devx** [comment]
Docs Build status updates of commit _[97805c8](https://github.com/OfficeDev/office-js-docs-reference/commits/97805c8111ea6f8804abbecbbd6eb5514a2945f3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fivearrowsgrayset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fivearrowsgrayset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fivearrowsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fivearrowsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fiveboxesset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fiveboxesset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fivequartersset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fivequartersset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fiveratingset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fiveratingset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fourarrowsgrayset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fourarrowsgrayset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fourarrowsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fourarrowsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fourratingset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fourratingset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fourredtoblackset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fourredtoblackset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fourtrafficlightsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fourtrafficlightsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.iconcollections.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.iconcollections.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.requestcontext.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.requestcontext.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threearrowsgrayset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threearrowsgrayset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threearrowsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threearrowsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threeflagsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threeflagsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threesignsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threesignsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threestarsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threestarsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threesymbols2set.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threesymbols2set.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threesymbolsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threesymbolsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threetrafficlights1set.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threetrafficlights1set.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threetrafficlights2set.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threetrafficlights2set.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threetrianglesset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threetrianglesset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_1/excel/excel.chart.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_1/excel/excel.chart.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_1/excel/excel.range.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_1/excel/excel.range.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_1/excel/excel.table.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_1/excel/excel.table.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_1/excel/excel.workbook.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_1/excel/excel.workbook.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_1/excel/excel.worksheet.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_1/excel/excel.worksheet.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.chart.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.chart.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fivearrowsgrayset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fivearrowsgrayset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fivearrowsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fivearrowsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fiveboxesset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fiveboxesset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fivequartersset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fivequartersset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fiveratingset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fiveratingset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fourarrowsgrayset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fourarrowsgrayset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fourarrowsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fourarrowsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fourratingset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fourratingset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fourredtoblackset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fourredtoblackset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fourtrafficlightsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fourtrafficlightsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.iconcollections.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.iconcollections.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.range.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.range.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.table.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.table.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.threearrowsgrayset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.threearrowsgrayset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.threearrowsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.threearrowsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.threeflagsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.threeflagsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.threesignsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.threesignsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.threestarsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.threestarsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.threesymbols2set.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.threesymbols2set.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.threesymbolsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.threesymbolsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.threetrafficlights1set.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.threetrafficlights1set.yml) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C14%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201908142152473025-486%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=un5D%2FKIeWMwPEq1CitIrolQqAG%2F1%2F%2Fi2VfjAbt%2F2XOA%3D&st=2019-08-14T22%3A16%3A44Z&se=2019-09-14T22%3A21%3A44Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/docs-ref-autogen/excel_1_1/excel/excel.chart.yml [hunk 2]
```diff
@@ -1,8 +1,10 @@
 ### YamlMime:UniversalReference
 items:
   - uid: excel.Excel.Chart
-    summary: |-
-      Represents a chart object in a workbook.
+    summary: >-
+      Represents a chart object in a workbook. To learn more about the Chart object model, see [Work with charts using
+      the Excel JavaScript API](https://docs.microsoft.com/office/dev/add-ins/excel/excel-add-ins-charts)<!-- -->.
```
Comment (ElizabethSamuel-MSFT) [inline docs/docs-ref-autogen/excel_1_1/excel/excel.chart.yml]: Do we have any concerns about articles like this using APIs that were introduced in later requirement sets?

File: generate-docs/tools/dts-diff.ts (hunk 1 above)
Comment (AlexJerabek) [reply generate-docs/tools/dts-diff.ts]: That was a separate tool I'm working on. Didn't mean to include that in the PR.

File: docs/docs-ref-autogen/excel_1_1/excel/excel.chart.yml (hunk 2 above)
Comment (AlexJerabek) [reply docs/docs-ref-autogen/excel_1_1/excel/excel.chart.yml]: I don't think so. We could consider versioned conceptual docs, but that feels like a lot of effort for a low impact scenario. Those Excel docs are also relatively scenario-driven, so if you're looking to work with PivotTables, you'll need the versions of those APIs.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[576bb33](https://github.com/OfficeDev/office-js-docs-reference/commits/576bb33ef7db1e7135b11c0a826c5f49b76a3220)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fivearrowsgrayset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fivearrowsgrayset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fivearrowsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fivearrowsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fiveboxesset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fiveboxesset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fivequartersset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fivequartersset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fiveratingset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fiveratingset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fourarrowsgrayset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fourarrowsgrayset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fourarrowsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fourarrowsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fourratingset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fourratingset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fourredtoblackset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fourredtoblackset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.fourtrafficlightsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.fourtrafficlightsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.iconcollections.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.iconcollections.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.requestcontext.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.requestcontext.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threearrowsgrayset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threearrowsgrayset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threearrowsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threearrowsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threeflagsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threeflagsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threesignsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threesignsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threestarsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threestarsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threesymbols2set.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threesymbols2set.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threesymbolsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threesymbolsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threetrafficlights1set.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threetrafficlights1set.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threetrafficlights2set.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threetrafficlights2set.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel/excel.threetrianglesset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel/excel/excel.threetrianglesset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_1/excel/excel.chart.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_1/excel/excel.chart.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_1/excel/excel.range.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_1/excel/excel.range.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_1/excel/excel.table.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_1/excel/excel.table.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_1/excel/excel.workbook.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_1/excel/excel.workbook.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_1/excel/excel.worksheet.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_1/excel/excel.worksheet.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.chart.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.chart.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fivearrowsgrayset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fivearrowsgrayset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fivearrowsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fivearrowsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fiveboxesset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fiveboxesset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fivequartersset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fivequartersset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fiveratingset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fiveratingset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fourarrowsgrayset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fourarrowsgrayset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fourarrowsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fourarrowsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fourratingset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fourratingset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fourredtoblackset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fourredtoblackset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.fourtrafficlightsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.fourtrafficlightsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.iconcollections.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.iconcollections.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.range.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.range.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.table.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.table.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.threearrowsgrayset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.threearrowsgrayset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.threearrowsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.threearrowsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.threeflagsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.threeflagsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.threesignsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.threesignsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.threestarsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.threestarsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.threesymbols2set.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.threesymbols2set.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.threesymbolsset.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.threesymbolsset.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel_1_2/excel/excel.threetrafficlights1set.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/AlexJ-DiffTool/docs/docs-ref-autogen/excel_1_2/excel/excel.threetrafficlights1set.yml) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C15%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201908152105479708-486%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=kXIFfQ6UcTdbZD6ZdeSOLI0F2V57LbP0KU2qoMyh2v8%3D&st=2019-08-15T21%3A20%3A32Z&se=2019-09-15T21%3A25%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**